cmd/kaf/kaf.go

```diff
@@ -29,6 +29,41 @@ func getConfig() (saramaConfig *sarama.Config) {
 		saramaConfig.Net.SASL.User = cluster.SASL.Username
 		saramaConfig.Net.SASL.Password = cluster.SASL.Password
 	}
+	if cluster.TLS != nil && cluster.SecurityProtocol != "SASL_SSL" {
+		saramaConfig.Net.TLS.Enable = true
+		tlsConfig := &tls.Config{
+			InsecureSkipVerify: cluster.TLS.Insecure,
+		}
+
+		caCert, err := ioutil.ReadFile(cluster.TLS.Cafile)
```

Comment:
can we make the CA file optional? since we allow insecure if the flag in the config for insecure it set, we don't need to enforce the cert in this case. so: allow not having a cacert if the insecure flag is set.

Reply:
if we enable `InsecureSkipVerify`, the client will not verify the certificate presented by the server (namely, check that the hostname matches):

> InsecureSkipVerify controls whether a client verifies server's certificate chain and host name.

By including the CA file, we permit the client to check that the certificate presented _is_ signed by the CA we have.

Reply:
as far as i can see, your code would return an error if the caFile file does not exist. even if Insecure is enabled (and hence it would not need the ca cert). that's why i'm asking if we can make the "loading of the ca file" optional if insecure is enabled.

Reply:
note. thanks for the feedback!